src/views/Leaderboard.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+  <div class="container">
+    <h1>Leaderboard</h1>
+    <div class="board">
+      <!-- heading to select view -->
+      <div class="heading">
+        <button class="active" @click="openPlayers" v-if="showPlayers">Players</button>
+        <button class="inactive" @click="openPlayers" v-if="!showPlayers">Players</button>
+        <div class=divider></div>
+        <button class="active" @click="openProgress" v-if="!showPlayers">My progress</button>
+        <button class="inactive" @click="openProgress" v-if="showPlayers">My progress</button>
+      </div>
+      <!-- Players -->
+      <div class="rows" v-if="showPlayers">
+        <!-- replace with v-for once data is loaded in -->
```

Comment:
You should create a separate component for each of the Players and My Progress pages. Generally, we want to always break down the page into the smallest pieces we can.